### DISCUSSION TITLE
🚨Can only connect with DSMR-5S

### DISCUSSION THREAD

**Discussion by spafruit1** [dsmrreader/dsmr-reader#1608] [Questions & answers / How-to's] [answered]
### Description

My P1 reader was working fine with DSMR 5 untill it stopped working a week. Now i can only connect with DSMR 5S but i get no data i got only 0 value's :( The P1 reader is stil reachable with telnet. 

Any suggetions on what i can try next?

### DSMR-reader version

5 and 5S

**Comment by dennissiemensma** [comment]
Hi I'm not sure what 5s is. Did you try using the cu program as well? If it suddenly broke down and cu does not work either, there is a high possibility that the reader or cable stopped working.

**Comment by spafruit1** [comment]
5s means DSMR version 5s (s means Swedish but mine is dutch). What is the cu program? I am able to have communication via CMD telnet
![IMG_20220318_191411](https://user-images.githubusercontent.com/35542961/159136553-d8028956-aa46-4fbe-8a12-d780614aabbe.jpg)
![dsmr version](https://user-images.githubusercontent.com/35542961/159136634-e867ec2e-3745-409d-a6bc-e2e1876d327e.png)

**Comment by dennissiemensma** [comment]
The cu program can be used to read serial cables. If you're not using a cable but some other appliance, it should be fine without. Using telnet or cu allows you to check what dsmrreader reads. It seems your telegram has some corruption with weird symbols. I'm not sure if 5s uses other symbols, but dsmrreader does not support these for sure. You may want to check for any signal interference causing these issues.

**Comment by spafruit1** [comment]
I got this weird symbols in both 5 and 5s. Maybe that is my problem i go try a new cable and let you know

**Comment by spafruit1** [comment, marked as answer]
Update: It turned out that i had a bad P1 reader. After switching it for a new one everything was working fine again :)

**Reply by dennissiemensma** [reply]
Great to hear, thanks for providing the update!
